File: src/test/java/SortCodeRangeTests.java
import com.github.pauldambra.moduluschecker.valacdosFile.SortCodeRange;
import org.junit.Test;

import static org.hamcrest.CoreMatchers.is;
import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.core.IsEqual.equalTo;

public class SortCodeRangeTests {
    @Test
    public void SortCodeRangeCanContain() {
        SortCodeRange range = new SortCodeRange("010004", "010006");
        assertThat(range.contains("010003"), is(equalTo(false)));
        assertThat(range.contains("010004"), is(equalTo(true)));
        assertThat(range.contains("010005"), is(equalTo(true)));
        assertThat(range.contains("010006"), is(equalTo(true)));
        assertThat(range.contains("010007"), is(equalTo(false)));
    }
}

File: src/test/java/SortCodeRangeTests.kt


import com.github.pauldambra.moduluschecker.valacdosFile.SortCodeRange
import com.natpryce.hamkrest.assertion.assert
import com.natpryce.hamkrest.equalTo
import org.junit.Test

class SortCodeRangeTests {
    @Test
    fun sortCodeRangeCanContain() {
        val range = SortCodeRange("010004", "010006")
        assert.that(range.contains("010003"), equalTo(false))
        assert.that(range.contains("010004"), equalTo(true))
        assert.that(range.contains("010005"), equalTo(true))
        assert.that(range.contains("010006"), equalTo(true))
        assert.that(range.contains("010007"), equalTo(false))
    }
}
